src/pages/_app.page.tsx

```diff
@@ -14,6 +14,7 @@ import { joyIdConfig } from 'core/joyid/config';
 import Head from 'next/head';
 import theme from 'constants/theme';
 
+
```

Comment:
To resolve - why could you not import bitcoin connect here? it's needed so zaps work on every page

Reply:
@Alan-AC7  For NextJS you can add this to `src/components/PostItems/PostReactions/index.tsx` to only import the library clientside:

```
useEffect(() => {
    // load bitcoin connect for zaps
    import("@getalby/bitcoin-connect-react");
  }, []);
 ```